### ISSUE TITLE
The extension doesn't build on iOS since 17.07.2023

### ISSUE THREAD

**Issue by Morgerion** [defold/extension-facebook#53]
It is not possible to build a project on iOS with this extension if the project uses [Admob ](https://github.com/defold/extension-admob/) or [Ironsource](https://github.com/defold/extension-ironsource) or [AppsFlyer](https://github.com/SalavatR/def_appsflyer_ad_revenue/) extension.

Here is the log file when there is a conflict with **Ironsource**:
[log.txt](https://github.com/defold/extension-facebook/files/12074534/log.txt)

**Comment by britzl** [comment]
@Morgerion thank you for the bug report. I will look into this today. Can you please tell me which versions of the extensions you are using?

**Comment by britzl** [comment]
I tried Defold 1.4.8 with https://build.defold.com and the latest versions of extension-facebook (7.1.1) and extension-admob (3.1.2) and was able to build an IPA:
<img width="834" alt="Screenshot 2023-07-18 at 10 33 35" src="https://github.com/defold/extension-facebook/assets/1300688/7c71809b-616f-4a7b-a590-7a7ced916ad7">

**Comment by AGulev** [comment]
@britzl pls try the latest ironsource and the latest facobook
make sure ironsource includes FB adapter

**Comment by britzl** [comment]
> @britzl pls try the latest ironsource and the latest facobook make sure ironsource includes FB adapter

Yes, it does not work with extension-ironsource and extension-facebook. I think we need to migrate extension-facebook to use Cocoapods (we have a branch).
